_posts/2020-01-21-200wad-logo.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: 200WaD - Logo
+date: 2020-01-21 16:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [Writing]
+image:
+image2:
+author: Valentino Urbano
+---
+
+Note: This post is about the new logo @ [200WaD][1]. If you're not using the website you might not be interested in it, feel free to skip it.
+
+I really liked the original 200 words a day logo (the old logo) even though baz said multiple times that he made it himself and didn't really take too much time with it. I consider that a plus since it came out really well despite that.
+
+Simple, but with a meaning behind it. Obvious to its meaning, but not too much to cheapen it and make it feel too basic.
+
+This new one feels that way to me. I agree that a change was needed since the rebranding, but I'm not a huge fan of the current choice.[^1] 
+
+I always found making good logos to be hard. A basic logo is really easy to make instead, but in most cases what you end up with is nothing more than "meh". It is ok and might even look good, but it doesn't really transmit what you want it to.
+
+To make a really good logo you need a lot of tries to really home in to the right design and that takes time.[^2] On the other hand, as with everything, perfectionists never ship anything if they don't tell themselves it is good enough. Time to put it out there.
+
+What is your process? Do you start with an idea and work on the logo from that idea? Do you take a basic shape and morph it to something more? 
```

Comment:
What do you do?

Reply:
I'm terrible at logos so I don't have a process :P